.github/workflows/test.yaml

```
@@ -0,0 +1,20 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - run: npm ci
+      - run: npm test
```

```suggestion
      - run: npm cit
```

from https://docs.npmjs.com/cli/v7/commands/npm-install-ci-test